package.json

```diff
@@ -430,6 +431,11 @@
         "category": "R",
         "command": "r.createLintrConfig"
       },
+      {
+        "title": "Generate 'c_cpp_properties.json'",
```

Comment:
One more thing, do you think the command title should contain a file name `c_cpp_properties.json`, (which users new to VS Code or C/C++ extension might not know what it is) or should we use how C/C++ extension calls it "C/C++ Configurations (JSON)"?

<img width="601" alt="image" src="https://user-images.githubusercontent.com/4662568/193323497-8974ee1f-1360-47b8-b49c-b495f5b7b890.png">

Reply:
Maybe "Generate C/C++ Configuration for R package" would be a good description?

Reply:
We have "R Package" category in `package.json`, it would be nice to have something like

```json
{
        "title": "Generate C/C++ Configuration",
        "category": "R Package",
        "command": "r.generateCCppProperties"
}
```

Reply:
I have renamed the command to your suggestion and moved the entry below to the other `"category": "R Package"` entries